src/styles/blocks/header.scss

```diff
@@ -27,9 +27,16 @@
     list-style: none;
     line-height: 60px;
 
+    transition: color $t-transition-time ease-in-out;
+
     &:not(:last-child) {
       margin-right: 20px;
     }
+
+    &:hover {
+      color: $c-blue-title-hover;
+      transform: color;
```

Comment:
I think this is redundant code  transform: color;

Reply:
Fix your mate's comment.